test/apis/user-api.ts

```diff
@@ -0,0 +1,46 @@
+import {getEnvVariables, TestUtils} from "../test-utils";
+import {v4 as uuid} from 'uuid';
+import {assert} from "chai";
+import 'isomorphic-fetch'
+import {tmpdir} from "os";
+import {TextDecoder, TextEncoder} from "util";
+
+(global as any).localStorage = new (require('node-localstorage').LocalStorage)(tmpdir(), 5 * 1024 * 1024 * 1024)
+;(global as any).fetch = fetch
+;(global as any).Storage = ''
+;(global as any).TextDecoder = TextDecoder
+;(global as any).TextEncoder = TextEncoder
+
+const {iCureUrl: iCureUrl, msgGtwUrl: msgGtwUrl, authProcessHcpId: authProcessHcpId, specId: specId} = getEnvVariables()
+
+describe('User API', () => {
+  it('Management of auto-delegations', async () => {
```

Comment:
Rename your test to `a user is able to share data with another dataOwner, and stop sharing data with him later`

Add the following test as well : 
- `When a user already shares data with the provided dataOwner, the user is returned successfully, without any additional request to iCure` and check that the user is exactly the same than the provided one